OasisDEX/oasis-borrow
# Manage GUNI Earn Details

# [Manage GUNI Earn Details](https://app.shortcut.com/oazo-apps/story/3489/manage-earn-position)

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/3783198/176488960-06caaa0b-53df-4056-908b-98d7be9c9452.png">


## Changes 👷‍♀️
  <Please add short list of changes>
- Implements the details page on Manage GUNI view.  Does not include any changes to the form.
  
## How to test 🧪
- load a position (you can see positions from here: https://tracker-vaults.makerdao.network/)
- ensure that the values are correct
